{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Creating ../results/skm01 ...\n",
      "[1] 6893\n",
      "[2] 6894\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "8876156 reads; of these:\n",
      "  8876156 (100.00%) were paired; of these:\n",
      "    1745965 (19.67%) aligned concordantly 0 times\n",
      "    6896409 (77.70%) aligned concordantly exactly 1 time\n",
      "    233782 (2.63%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1745965 pairs aligned concordantly 0 times; of these:\n",
      "      8715 (0.50%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1737250 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3474500 mates make up the pairs; of these:\n",
      "        2199743 (63.31%) aligned 0 times\n",
      "        1223275 (35.21%) aligned exactly 1 time\n",
      "        51482 (1.48%) aligned >1 times\n",
      "87.61% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm02 ...\n",
      "[1] 13911\n",
      "[2] 13912\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "22305849 reads; of these:\n",
      "  22305849 (100.00%) were paired; of these:\n",
      "    4129292 (18.51%) aligned concordantly 0 times\n",
      "    16855199 (75.56%) aligned concordantly exactly 1 time\n",
      "    1321358 (5.92%) aligned concordantly >1 times\n",
      "    ----\n",
      "    4129292 pairs aligned concordantly 0 times; of these:\n",
      "      25909 (0.63%) aligned discordantly 1 time\n",
      "    ----\n",
      "    4103383 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      8206766 mates make up the pairs; of these:\n",
      "        4943866 (60.24%) aligned 0 times\n",
      "        2993690 (36.48%) aligned exactly 1 time\n",
      "        269210 (3.28%) aligned >1 times\n",
      "88.92% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm03 ...\n",
      "[1] 31894\n",
      "[2] 31895\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "10601968 reads; of these:\n",
      "  10601968 (100.00%) were paired; of these:\n",
      "    2047249 (19.31%) aligned concordantly 0 times\n",
      "    6849276 (64.60%) aligned concordantly exactly 1 time\n",
      "    1705443 (16.09%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2047249 pairs aligned concordantly 0 times; of these:\n",
      "      39353 (1.92%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2007896 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4015792 mates make up the pairs; of these:\n",
      "        2403447 (59.85%) aligned 0 times\n",
      "        1271420 (31.66%) aligned exactly 1 time\n",
      "        340925 (8.49%) aligned >1 times\n",
      "88.67% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm04 ...\n",
      "[1] 8647\n",
      "[2] 8648\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "10004145 reads; of these:\n",
      "  10004145 (100.00%) were paired; of these:\n",
      "    2201869 (22.01%) aligned concordantly 0 times\n",
      "    7501898 (74.99%) aligned concordantly exactly 1 time\n",
      "    300378 (3.00%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2201869 pairs aligned concordantly 0 times; of these:\n",
      "      16155 (0.73%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2185714 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4371428 mates make up the pairs; of these:\n",
      "        2953373 (67.56%) aligned 0 times\n",
      "        1351964 (30.93%) aligned exactly 1 time\n",
      "        66091 (1.51%) aligned >1 times\n",
      "85.24% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm05 ...\n",
      "[1] 16600\n",
      "[2] 16601\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "8417221 reads; of these:\n",
      "  8417221 (100.00%) were paired; of these:\n",
      "    1571947 (18.68%) aligned concordantly 0 times\n",
      "    6340503 (75.33%) aligned concordantly exactly 1 time\n",
      "    504771 (6.00%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1571947 pairs aligned concordantly 0 times; of these:\n",
      "      13145 (0.84%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1558802 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3117604 mates make up the pairs; of these:\n",
      "        1889354 (60.60%) aligned 0 times\n",
      "        1124563 (36.07%) aligned exactly 1 time\n",
      "        103687 (3.33%) aligned >1 times\n",
      "88.78% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm06 ...\n",
      "[1] 23377\n",
      "[2] 23378\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "21975246 reads; of these:\n",
      "  21975246 (100.00%) were paired; of these:\n",
      "    4557729 (20.74%) aligned concordantly 0 times\n",
      "    13958157 (63.52%) aligned concordantly exactly 1 time\n",
      "    3459360 (15.74%) aligned concordantly >1 times\n",
      "    ----\n",
      "    4557729 pairs aligned concordantly 0 times; of these:\n",
      "      112815 (2.48%) aligned discordantly 1 time\n",
      "    ----\n",
      "    4444914 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      8889828 mates make up the pairs; of these:\n",
      "        5585951 (62.84%) aligned 0 times\n",
      "        2612929 (29.39%) aligned exactly 1 time\n",
      "        690948 (7.77%) aligned >1 times\n",
      "87.29% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm07 ...\n",
      "[1] 9252\n",
      "[2] 9253\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "15591498 reads; of these:\n",
      "  15591498 (100.00%) were paired; of these:\n",
      "    3043533 (19.52%) aligned concordantly 0 times\n",
      "    12148345 (77.92%) aligned concordantly exactly 1 time\n",
      "    399620 (2.56%) aligned concordantly >1 times\n",
      "    ----\n",
      "    3043533 pairs aligned concordantly 0 times; of these:\n",
      "      13085 (0.43%) aligned discordantly 1 time\n",
      "    ----\n",
      "    3030448 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      6060896 mates make up the pairs; of these:\n",
      "        3835931 (63.29%) aligned 0 times\n",
      "        2136498 (35.25%) aligned exactly 1 time\n",
      "        88467 (1.46%) aligned >1 times\n",
      "87.70% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm08 ...\n",
      "[1] 21597\n",
      "[2] 21598\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "10979331 reads; of these:\n",
      "  10979331 (100.00%) were paired; of these:\n",
      "    2113304 (19.25%) aligned concordantly 0 times\n",
      "    6027668 (54.90%) aligned concordantly exactly 1 time\n",
      "    2838359 (25.85%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2113304 pairs aligned concordantly 0 times; of these:\n",
      "      37011 (1.75%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2076293 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4152586 mates make up the pairs; of these:\n",
      "        2452503 (59.06%) aligned 0 times\n",
      "        1127803 (27.16%) aligned exactly 1 time\n",
      "        572280 (13.78%) aligned >1 times\n",
      "88.83% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm09 ...\n",
      "[1] 30963\n",
      "[2] 30964\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "12701529 reads; of these:\n",
      "  12701529 (100.00%) were paired; of these:\n",
      "    2295910 (18.08%) aligned concordantly 0 times\n",
      "    9325226 (73.42%) aligned concordantly exactly 1 time\n",
      "    1080393 (8.51%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2295910 pairs aligned concordantly 0 times; of these:\n",
      "      12055 (0.53%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2283855 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4567710 mates make up the pairs; of these:\n",
      "        2693635 (58.97%) aligned 0 times\n",
      "        1651862 (36.16%) aligned exactly 1 time\n",
      "        222213 (4.86%) aligned >1 times\n",
      "89.40% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm10 ...\n",
      "[1] 8746\n",
      "[2] 8747\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "4462649 reads; of these:\n",
      "  4462649 (100.00%) were paired; of these:\n",
      "    832520 (18.66%) aligned concordantly 0 times\n",
      "    3393178 (76.04%) aligned concordantly exactly 1 time\n",
      "    236951 (5.31%) aligned concordantly >1 times\n",
      "    ----\n",
      "    832520 pairs aligned concordantly 0 times; of these:\n",
      "      6159 (0.74%) aligned discordantly 1 time\n",
      "    ----\n",
      "    826361 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      1652722 mates make up the pairs; of these:\n",
      "        1005109 (60.82%) aligned 0 times\n",
      "        598602 (36.22%) aligned exactly 1 time\n",
      "        49011 (2.97%) aligned >1 times\n",
      "88.74% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm11 ...\n",
      "[1] 12346\n",
      "[2] 12347\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "7697062 reads; of these:\n",
      "  7697062 (100.00%) were paired; of these:\n",
      "    1454991 (18.90%) aligned concordantly 0 times\n",
      "    6040699 (78.48%) aligned concordantly exactly 1 time\n",
      "    201372 (2.62%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1454991 pairs aligned concordantly 0 times; of these:\n",
      "      6575 (0.45%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1448416 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      2896832 mates make up the pairs; of these:\n",
      "        1796729 (62.02%) aligned 0 times\n",
      "        1056957 (36.49%) aligned exactly 1 time\n",
      "        43146 (1.49%) aligned >1 times\n",
      "88.33% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm12 ...\n",
      "[1] 18598\n",
      "[2] 18599\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "27837579 reads; of these:\n",
      "  27837579 (100.00%) were paired; of these:\n",
      "    5682416 (20.41%) aligned concordantly 0 times\n",
      "    16489301 (59.23%) aligned concordantly exactly 1 time\n",
      "    5665862 (20.35%) aligned concordantly >1 times\n",
      "    ----\n",
      "    5682416 pairs aligned concordantly 0 times; of these:\n",
      "      129781 (2.28%) aligned discordantly 1 time\n",
      "    ----\n",
      "    5552635 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      11105270 mates make up the pairs; of these:\n",
      "        6837259 (61.57%) aligned 0 times\n",
      "        3132357 (28.21%) aligned exactly 1 time\n",
      "        1135654 (10.23%) aligned >1 times\n",
      "87.72% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm13 ...\n",
      "[1] 8355\n",
      "[2] 8356\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "7712017 reads; of these:\n",
      "  7712017 (100.00%) were paired; of these:\n",
      "    1477111 (19.15%) aligned concordantly 0 times\n",
      "    6015112 (78.00%) aligned concordantly exactly 1 time\n",
      "    219794 (2.85%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1477111 pairs aligned concordantly 0 times; of these:\n",
      "      9149 (0.62%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1467962 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      2935924 mates make up the pairs; of these:\n",
      "        1839709 (62.66%) aligned 0 times\n",
      "        1048911 (35.73%) aligned exactly 1 time\n",
      "        47304 (1.61%) aligned >1 times\n",
      "88.07% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm14 ...\n",
      "[1] 14390\n",
      "[2] 14391\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "18792805 reads; of these:\n",
      "  18792805 (100.00%) were paired; of these:\n",
      "    3311361 (17.62%) aligned concordantly 0 times\n",
      "    14725648 (78.36%) aligned concordantly exactly 1 time\n",
      "    755796 (4.02%) aligned concordantly >1 times\n",
      "    ----\n",
      "    3311361 pairs aligned concordantly 0 times; of these:\n",
      "      16901 (0.51%) aligned discordantly 1 time\n",
      "    ----\n",
      "    3294460 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      6588920 mates make up the pairs; of these:\n",
      "        3842229 (58.31%) aligned 0 times\n",
      "        2574535 (39.07%) aligned exactly 1 time\n",
      "        172156 (2.61%) aligned >1 times\n",
      "89.78% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm15 ...\n",
      "[1] 29271\n",
      "[2] 29272\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "51015759 reads; of these:\n",
      "  51015759 (100.00%) were paired; of these:\n",
      "    9184567 (18.00%) aligned concordantly 0 times\n",
      "    39078544 (76.60%) aligned concordantly exactly 1 time\n",
      "    2752648 (5.40%) aligned concordantly >1 times\n",
      "    ----\n",
      "    9184567 pairs aligned concordantly 0 times; of these:\n",
      "      94125 (1.02%) aligned discordantly 1 time\n",
      "    ----\n",
      "    9090442 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      18180884 mates make up the pairs; of these:\n",
      "        10670688 (58.69%) aligned 0 times\n",
      "        6928079 (38.11%) aligned exactly 1 time\n",
      "        582117 (3.20%) aligned >1 times\n",
      "89.54% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm16 ...\n",
      "[1] 5474\n",
      "[2] 5475\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "9535029 reads; of these:\n",
      "  9535029 (100.00%) were paired; of these:\n",
      "    2013777 (21.12%) aligned concordantly 0 times\n",
      "    7266444 (76.21%) aligned concordantly exactly 1 time\n",
      "    254808 (2.67%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2013777 pairs aligned concordantly 0 times; of these:\n",
      "      20055 (1.00%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1993722 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3987444 mates make up the pairs; of these:\n",
      "        2621190 (65.74%) aligned 0 times\n",
      "        1301492 (32.64%) aligned exactly 1 time\n",
      "        64762 (1.62%) aligned >1 times\n",
      "86.25% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm17 ...\n",
      "[1] 12944\n",
      "[2] 12945\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "6619731 reads; of these:\n",
      "  6619731 (100.00%) were paired; of these:\n",
      "    1601439 (24.19%) aligned concordantly 0 times\n",
      "    4752321 (71.79%) aligned concordantly exactly 1 time\n",
      "    265971 (4.02%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1601439 pairs aligned concordantly 0 times; of these:\n",
      "      7466 (0.47%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1593973 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3187946 mates make up the pairs; of these:\n",
      "        2266140 (71.08%) aligned 0 times\n",
      "        845365 (26.52%) aligned exactly 1 time\n",
      "        76441 (2.40%) aligned >1 times\n",
      "82.88% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm18 ...\n",
      "[1] 18249\n",
      "[2] 18250\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "8963217 reads; of these:\n",
      "  8963217 (100.00%) were paired; of these:\n",
      "    2022220 (22.56%) aligned concordantly 0 times\n",
      "    5999621 (66.94%) aligned concordantly exactly 1 time\n",
      "    941376 (10.50%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2022220 pairs aligned concordantly 0 times; of these:\n",
      "      38065 (1.88%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1984155 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3968310 mates make up the pairs; of these:\n",
      "        2657844 (66.98%) aligned 0 times\n",
      "        1108728 (27.94%) aligned exactly 1 time\n",
      "        201738 (5.08%) aligned >1 times\n",
      "85.17% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm19 ...\n",
      "[1] 25231\n",
      "[2] 25232\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "6355585 reads; of these:\n",
      "  6355585 (100.00%) were paired; of these:\n",
      "    1288040 (20.27%) aligned concordantly 0 times\n",
      "    4897271 (77.05%) aligned concordantly exactly 1 time\n",
      "    170274 (2.68%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1288040 pairs aligned concordantly 0 times; of these:\n",
      "      6364 (0.49%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1281676 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      2563352 mates make up the pairs; of these:\n",
      "        1670123 (65.15%) aligned 0 times\n",
      "        855296 (33.37%) aligned exactly 1 time\n",
      "        37933 (1.48%) aligned >1 times\n",
      "86.86% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm20 ...\n",
      "[1] 30270\n",
      "[2] 30271\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "5829529 reads; of these:\n",
      "  5829529 (100.00%) were paired; of these:\n",
      "    1053329 (18.07%) aligned concordantly 0 times\n",
      "    4538543 (77.85%) aligned concordantly exactly 1 time\n",
      "    237657 (4.08%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1053329 pairs aligned concordantly 0 times; of these:\n",
      "      6236 (0.59%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1047093 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      2094186 mates make up the pairs; of these:\n",
      "        1242517 (59.33%) aligned 0 times\n",
      "        797116 (38.06%) aligned exactly 1 time\n",
      "        54553 (2.60%) aligned >1 times\n",
      "89.34% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm21 ...\n",
      "[1] 2805\n",
      "[2] 2806\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "7736755 reads; of these:\n",
      "  7736755 (100.00%) were paired; of these:\n",
      "    1635086 (21.13%) aligned concordantly 0 times\n",
      "    4826897 (62.39%) aligned concordantly exactly 1 time\n",
      "    1274772 (16.48%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1635086 pairs aligned concordantly 0 times; of these:\n",
      "      15625 (0.96%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1619461 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3238922 mates make up the pairs; of these:\n",
      "        2088881 (64.49%) aligned 0 times\n",
      "        883365 (27.27%) aligned exactly 1 time\n",
      "        266676 (8.23%) aligned >1 times\n",
      "86.50% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm22 ...\n",
      "[1] 8954\n",
      "[2] 8955\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "7031218 reads; of these:\n",
      "  7031218 (100.00%) were paired; of these:\n",
      "    1553199 (22.09%) aligned concordantly 0 times\n",
      "    5288553 (75.22%) aligned concordantly exactly 1 time\n",
      "    189466 (2.69%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1553199 pairs aligned concordantly 0 times; of these:\n",
      "      10424 (0.67%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1542775 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3085550 mates make up the pairs; of these:\n",
      "        2082957 (67.51%) aligned 0 times\n",
      "        955882 (30.98%) aligned exactly 1 time\n",
      "        46711 (1.51%) aligned >1 times\n",
      "85.19% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm23 ...\n",
      "[1] 14748\n",
      "[2] 14749\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "235 reads; of these:\n",
      "  235 (100.00%) were paired; of these:\n",
      "    183 (77.87%) aligned concordantly 0 times\n",
      "    52 (22.13%) aligned concordantly exactly 1 time\n",
      "    0 (0.00%) aligned concordantly >1 times\n",
      "    ----\n",
      "    183 pairs aligned concordantly 0 times; of these:\n",
      "      0 (0.00%) aligned discordantly 1 time\n",
      "    ----\n",
      "    183 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      366 mates make up the pairs; of these:\n",
      "        343 (93.72%) aligned 0 times\n",
      "        22 (6.01%) aligned exactly 1 time\n",
      "        1 (0.27%) aligned >1 times\n",
      "27.02% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm24 ...\n",
      "[1] 14830\n",
      "[2] 14831\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "7972092 reads; of these:\n",
      "  7972092 (100.00%) were paired; of these:\n",
      "    2044801 (25.65%) aligned concordantly 0 times\n",
      "    3898344 (48.90%) aligned concordantly exactly 1 time\n",
      "    2028947 (25.45%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2044801 pairs aligned concordantly 0 times; of these:\n",
      "      23984 (1.17%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2020817 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4041634 mates make up the pairs; of these:\n",
      "        2863255 (70.84%) aligned 0 times\n",
      "        750082 (18.56%) aligned exactly 1 time\n",
      "        428297 (10.60%) aligned >1 times\n",
      "82.04% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm25 ...\n",
      "[1] 21470\n",
      "[2] 21471\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "5933216 reads; of these:\n",
      "  5933216 (100.00%) were paired; of these:\n",
      "    1228031 (20.70%) aligned concordantly 0 times\n",
      "    4552939 (76.74%) aligned concordantly exactly 1 time\n",
      "    152246 (2.57%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1228031 pairs aligned concordantly 0 times; of these:\n",
      "      16616 (1.35%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1211415 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      2422830 mates make up the pairs; of these:\n",
      "        1566220 (64.64%) aligned 0 times\n",
      "        821245 (33.90%) aligned exactly 1 time\n",
      "        35365 (1.46%) aligned >1 times\n",
      "86.80% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm26 ...\n",
      "[1] 26271\n",
      "[2] 26272\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "10580363 reads; of these:\n",
      "  10580363 (100.00%) were paired; of these:\n",
      "    2196276 (20.76%) aligned concordantly 0 times\n",
      "    8120305 (76.75%) aligned concordantly exactly 1 time\n",
      "    263782 (2.49%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2196276 pairs aligned concordantly 0 times; of these:\n",
      "      18763 (0.85%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2177513 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4355026 mates make up the pairs; of these:\n",
      "        2845990 (65.35%) aligned 0 times\n",
      "        1442379 (33.12%) aligned exactly 1 time\n",
      "        66657 (1.53%) aligned >1 times\n",
      "86.55% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm27 ...\n",
      "[1] 2524\n",
      "[2] 2525\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "11283011 reads; of these:\n",
      "  11283011 (100.00%) were paired; of these:\n",
      "    2103195 (18.64%) aligned concordantly 0 times\n",
      "    8897931 (78.86%) aligned concordantly exactly 1 time\n",
      "    281885 (2.50%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2103195 pairs aligned concordantly 0 times; of these:\n",
      "      8689 (0.41%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2094506 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4189012 mates make up the pairs; of these:\n",
      "        2580355 (61.60%) aligned 0 times\n",
      "        1548224 (36.96%) aligned exactly 1 time\n",
      "        60433 (1.44%) aligned >1 times\n",
      "88.57% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm28 ...\n",
      "[1] 11594\n",
      "[2] 11595\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "38012513 reads; of these:\n",
      "  38012513 (100.00%) were paired; of these:\n",
      "    20586241 (54.16%) aligned concordantly 0 times\n",
      "    16893523 (44.44%) aligned concordantly exactly 1 time\n",
      "    532749 (1.40%) aligned concordantly >1 times\n",
      "    ----\n",
      "    20586241 pairs aligned concordantly 0 times; of these:\n",
      "      8238 (0.04%) aligned discordantly 1 time\n",
      "    ----\n",
      "    20578003 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      41156006 mates make up the pairs; of these:\n",
      "        38010625 (92.36%) aligned 0 times\n",
      "        3020627 (7.34%) aligned exactly 1 time\n",
      "        124754 (0.30%) aligned >1 times\n",
      "50.00% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm29 ...\n",
      "[1] 6374\n",
      "[2] 6375\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "11945297 reads; of these:\n",
      "  11945297 (100.00%) were paired; of these:\n",
      "    2306430 (19.31%) aligned concordantly 0 times\n",
      "    9350175 (78.27%) aligned concordantly exactly 1 time\n",
      "    288692 (2.42%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2306430 pairs aligned concordantly 0 times; of these:\n",
      "      4652 (0.20%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2301778 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4603556 mates make up the pairs; of these:\n",
      "        2939314 (63.85%) aligned 0 times\n",
      "        1602745 (34.82%) aligned exactly 1 time\n",
      "        61497 (1.34%) aligned >1 times\n",
      "87.70% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm30 ...\n",
      "[1] 15987\n",
      "[2] 15988\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "13013527 reads; of these:\n",
      "  13013527 (100.00%) were paired; of these:\n",
      "    2445922 (18.80%) aligned concordantly 0 times\n",
      "    10248480 (78.75%) aligned concordantly exactly 1 time\n",
      "    319125 (2.45%) aligned concordantly >1 times\n",
      "    ----\n",
      "    2445922 pairs aligned concordantly 0 times; of these:\n",
      "      7270 (0.30%) aligned discordantly 1 time\n",
      "    ----\n",
      "    2438652 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      4877304 mates make up the pairs; of these:\n",
      "        3022961 (61.98%) aligned 0 times\n",
      "        1785339 (36.61%) aligned exactly 1 time\n",
      "        69004 (1.41%) aligned >1 times\n",
      "88.39% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm31 ...\n",
      "[1] 25973\n",
      "[2] 25974\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "10542760 reads; of these:\n",
      "  10542760 (100.00%) were paired; of these:\n",
      "    1864133 (17.68%) aligned concordantly 0 times\n",
      "    8424274 (79.91%) aligned concordantly exactly 1 time\n",
      "    254353 (2.41%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1864133 pairs aligned concordantly 0 times; of these:\n",
      "      3820 (0.20%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1860313 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3720626 mates make up the pairs; of these:\n",
      "        2202554 (59.20%) aligned 0 times\n",
      "        1462741 (39.31%) aligned exactly 1 time\n",
      "        55331 (1.49%) aligned >1 times\n",
      "89.55% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm32 ...\n",
      "[1] 2283\n",
      "[2] 2284\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "10349064 reads; of these:\n",
      "  10349064 (100.00%) were paired; of these:\n",
      "    1907230 (18.43%) aligned concordantly 0 times\n",
      "    8188536 (79.12%) aligned concordantly exactly 1 time\n",
      "    253298 (2.45%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1907230 pairs aligned concordantly 0 times; of these:\n",
      "      3852 (0.20%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1903378 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3806756 mates make up the pairs; of these:\n",
      "        2327119 (61.13%) aligned 0 times\n",
      "        1420170 (37.31%) aligned exactly 1 time\n",
      "        59467 (1.56%) aligned >1 times\n",
      "88.76% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm33 ...\n",
      "[1] 10571\n",
      "[2] 10572\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "10235290 reads; of these:\n",
      "  10235290 (100.00%) were paired; of these:\n",
      "    1840673 (17.98%) aligned concordantly 0 times\n",
      "    8146892 (79.60%) aligned concordantly exactly 1 time\n",
      "    247725 (2.42%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1840673 pairs aligned concordantly 0 times; of these:\n",
      "      3462 (0.19%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1837211 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3674422 mates make up the pairs; of these:\n",
      "        2207964 (60.09%) aligned 0 times\n",
      "        1412709 (38.45%) aligned exactly 1 time\n",
      "        53749 (1.46%) aligned >1 times\n",
      "89.21% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n",
      "Creating ../results/skm34 ...\n",
      "[1] 18583\n",
      "[2] 18584\n",
      "[samopen] SAM header is present: 1 sequences.\n",
      "9058947 reads; of these:\n",
      "  9058947 (100.00%) were paired; of these:\n",
      "    1642725 (18.13%) aligned concordantly 0 times\n",
      "    7196717 (79.44%) aligned concordantly exactly 1 time\n",
      "    219505 (2.42%) aligned concordantly >1 times\n",
      "    ----\n",
      "    1642725 pairs aligned concordantly 0 times; of these:\n",
      "      3639 (0.22%) aligned discordantly 1 time\n",
      "    ----\n",
      "    1639086 pairs aligned 0 times concordantly or discordantly; of these:\n",
      "      3278172 mates make up the pairs; of these:\n",
      "        1983741 (60.51%) aligned 0 times\n",
      "        1242045 (37.89%) aligned exactly 1 time\n",
      "        52386 (1.60%) aligned >1 times\n",
      "89.05% overall alignment rate\n",
      "[1]-  Done                    zcat $sample_dir/R1.fastq.gz > $read1\n",
      "[2]+  Done                    zcat $sample_dir/R2.fastq.gz > $read2\n"
     ]
    }
   ],
   "source": [
    "index=\"../ref/ecoli/MG1655\"\n",
    "\n",
    "base_dir=\"../data\"\n",
    "\n",
    "for sample in $(ls $base_dir | grep skm)\n",
    "do\n",
    "    sample_dir=$base_dir/$sample\n",
    "    result_dir=\"../results/$sample\"\n",
    "    \n",
    "    if [ ! -d \"$result_dir\" ]; then\n",
    "        echo \"Creating $result_dir ...\"\n",
    "        mkdir -p $result_dir\n",
    "    fi\n",
    "    \n",
    "    read1=$sample_dir/read1.fifo\n",
    "    read2=$sample_dir/read2.fifo\n",
    "    mkfifo $read1\n",
    "    mkfifo $read2\n",
    "    \n",
    "    zcat $sample_dir/R1.fastq.gz > $read1 &\n",
    "    zcat $sample_dir/R2.fastq.gz > $read2 &\n",
    "    \n",
    "    # Align using bowtie2\n",
    "    bowtie2 -p 30 --very-sensitive -x $index \\\n",
    "        -1 $read1 -2 $read2 \\\n",
    "        | samtools view -bhS - > $result_dir/$sample.bam\n",
    "    \n",
    "    rm $sample_dir/*.fifo\n",
    "    \n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 1523\n",
      "[2] 1525\n",
      "[3] 1526\n",
      "[4] 1528\n",
      "[5] 1530\n",
      "[6] 1531\n",
      "[7] 1532\n",
      "[8] 1533\n",
      "[9] 1534\n",
      "[10] 1535\n",
      "[11] 1536\n",
      "[12] 1537\n",
      "[13] 1538\n",
      "[14] 1539\n",
      "[15] 1541\n",
      "[16] 1542\n",
      "[17] 1545\n",
      "[18] 1547\n",
      "[19] 1548\n",
      "[20] 1549\n",
      "[21] 1550\n",
      "[22] 1551\n",
      "[23] 1552\n",
      "[24] 1553\n",
      "[25] 1554\n",
      "[26] 1555\n",
      "[27] 1556\n",
      "[28] 1557\n",
      "[29] 1558\n",
      "[30] 1559\n",
      "[31] 1560\n",
      "[32] 1561\n",
      "[33] 1564\n",
      "[34] 1565\n"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# Sorting .bam files\n",
    "\n",
    "base_dir=\"../data\"\n",
    "\n",
    "for sample in $(ls $base_dir | grep skm)\n",
    "do\n",
    "    result_dir=\"../results/$sample\"\n",
    "    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted &\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files...\n",
      "[bam_sort_core] merging from 5 files...\n",
      "[bam_sort_core] merging from 5 files...\n",
      "[bam_sort_core] merging from 6 files...\n",
      "[bam_sort_core] merging from 6 files...\n",
      "[bam_sort_core] merging from 6 files...\n",
      "[bam_sort_core] merging from 7 files...\n",
      "[bam_sort_core] merging from 7 files...\n",
      "[bam_sort_core] merging from 7 files...\n",
      "[bam_sort_core] merging from 8 files...\n",
      "[bam_sort_core] merging from 8 files...\n",
      "[bam_sort_core] merging from 7 files...\n",
      "[bam_sort_core] merging from 8 files...\n",
      "[bam_sort_core] merging from 8 files...\n",
      "[bam_sort_core] merging from 9 files...\n",
      "[bam_sort_core] merging from 9 files...\n",
      "[bam_sort_core] merging from 9 files...\n",
      "[bam_sort_core] merging from 9 files...\n",
      "[bam_sort_core] merging from 10 files...\n",
      "[bam_sort_core] merging from 10 files...\n",
      "[bam_sort_core] merging from 9 files...\n",
      "[bam_sort_core] merging from 9 files...\n",
      "[bam_sort_core] merging from 9 files...\n",
      "[bam_sort_core] merging from 11 files...\n",
      "[bam_sort_core] merging from 7 files...\n",
      "[bam_sort_core] merging from 13 files...\n",
      "[bam_sort_core] merging from 11 files...\n",
      "[bam_sort_core] merging from 18 files...\n",
      "[bam_sort_core] merging from 19 files...\n",
      "[bam_sort_core] merging from 16 files...\n",
      "[bam_sort_core] merging from 31 files...\n",
      "[bam_sort_core] merging from 23 files...\n",
      "[bam_sort_core] merging from 42 files...\n",
      "[1]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[2]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[3]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[4]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[5]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[6]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[7]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[8]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[9]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[10]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[11]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[12]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[13]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[14]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[15]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[16]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[17]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[18]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[19]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[20]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[21]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[22]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[24]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[25]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[26]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[27]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[28]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[29]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[30]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[31]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[32]   Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[33]-  Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[34]+  Done                    samtools sort $result_dir/${sample}.bam $result_dir/${sample}_sorted\n",
      "[1] 25636\n",
      "[2] 25637\n",
      "[3] 25638\n",
      "[4] 25639\n",
      "[5] 25640\n",
      "[6] 25641\n",
      "[7] 25642\n",
      "[8] 25643\n",
      "[9] 25644\n",
      "[10] 25645\n",
      "[11] 25646\n",
      "[12] 25647\n",
      "[13] 25648\n",
      "[14] 25649\n",
      "[15] 25650\n",
      "[16] 25651\n",
      "[17] 25652\n",
      "[18] 25653\n",
      "[19] 25654\n",
      "[20] 25655\n",
      "[21] 25656\n",
      "[22] 25657\n",
      "[23] 25658\n",
      "[24] 25659\n",
      "[25] 25660\n",
      "[26] 25661\n",
      "[27] 25662\n",
      "[28] 25663\n",
      "[29] 25664\n",
      "[30] 25665\n",
      "[31] 25666\n",
      "[32] 25667\n",
      "[33] 25668\n",
      "[34] 25669\n"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# index .bam files\n",
    "\n",
    "for sample in $(ls $base_dir | grep skm)\n",
    "do\n",
    "    result_dir=\"../results/$sample\"\n",
    "    samtools index $result_dir/${sample}_sorted.bam &\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Bash",
   "language": "bash",
   "name": "bash"
  },
  "language_info": {
   "codemirror_mode": "shell",
   "file_extension": ".sh",
   "mimetype": "text/x-sh",
   "name": "bash"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
