{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from __future__ import division, print_function, unicode_literals"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 144K\r\n",
      "drwxr-xr-x 36 ilya lab  4.0K Oct 24 23:00 .\r\n",
      "drwxr-xr-x  6 ilya lab  4.0K Oct 24 18:39 ..\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm01\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:03 skm02\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm03\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm04\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm05\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:03 skm06\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:03 skm07\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm08\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm09\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm10\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm11\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:03 skm12\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm13\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:03 skm14\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:03 skm15\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm16\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm17\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm18\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm19\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm20\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm21\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm22\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm23\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm24\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm25\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm26\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm27\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:03 skm28\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm29\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm30\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm31\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm32\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm33\r\n",
      "drwxr-xr-x  2 ilya ilya 4.0K Oct 25 09:02 skm34\r\n"
     ]
    }
   ],
   "source": [
    "!ls -lah ../results/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "samples = [\n",
    "    {'id': 'skm01', 'Ab': 'FLAG', 'control': 'skm27', 'name': 'HepFLAG-WTS1-FLAG'},\n",
    "    {'id': 'skm02', 'Ab': 'RNAP', 'control': 'skm27', 'name': 'HepFLAG-WTS1-RNAP'},\n",
    "    {'id': 'skm03', 'Ab': 'S9.6', 'control': 'skm27', 'name': 'HepFLAG-WTS1-S9.6'},\n",
    "\n",
    "    {'id': 'skm04', 'Ab': 'FLAG', 'control': 'skm28', 'name': 'HepFLAG-S1d56-FLAG'},\n",
    "    {'id': 'skm05', 'Ab': 'RNAP', 'control': 'skm28', 'name': 'HepFLAG-S1d56-RNAP'},\n",
    "    {'id': 'skm06', 'Ab': 'S9.6', 'control': 'skm28', 'name': 'HepFLAG-S1d56-S9.6'},\n",
    "    \n",
    "    {'id': 'skm07', 'Ab': 'FLAG', 'control': 'skm28', 'name': 'S1d56-HepFLAG-FLAG'},\n",
    "    {'id': 'skm08', 'Ab': 'RNAP', 'control': 'skm28', 'name': 'S1d56-HepFLAG-RNAP'},\n",
    "    {'id': 'skm09', 'Ab': 'S9.6', 'control': 'skm28', 'name': 'S1d56-HepFLAG-S9.6'},\n",
    "    \n",
    "    {'id': 'skm10', 'Ab': 'FLAG', 'control': 'skm30', 'name': 'S1FLAG-dHep-FLAG'},\n",
    "    {'id': 'skm11', 'Ab': 'RNAP', 'control': 'skm30', 'name': 'S1FLAG-dHep-RNAP'},\n",
    "    {'id': 'skm12', 'Ab': 'S9.6', 'control': 'skm30', 'name': 'S1FLAG-dHep-S9.6'},\n",
    "    \n",
    "    {'id': 'skm13', 'Ab': 'FLAG', 'control': 'skm25', 'name': 'Hep-FLAG_V-FLAG'},\n",
    "    {'id': 'skm14', 'Ab': 'RNAP', 'control': 'skm25', 'name': 'Hep-FLAG_V-RNAP'},\n",
    "    {'id': 'skm15', 'Ab': 'S9.6', 'control': 'skm25', 'name': 'Hep-FLAG_V-S9.6'},\n",
    "    \n",
    "    {'id': 'skm16', 'Ab': 'FLAG', 'control': 'skm26', 'name': 'Hep-FLAG_rnh-FLAG'},\n",
    "    {'id': 'skm17', 'Ab': 'RNAP', 'control': 'skm26', 'name': 'Hep-FLAG_rnh-RNAP'},\n",
    "    {'id': 'skm18', 'Ab': 'S9.6', 'control': 'skm26', 'name': 'Hep-FLAG_rnh-S9.6'},\n",
    "    \n",
    "    {'id': 'skm19', 'Ab': 'FLAG', 'control': 'skm25', 'name': 'S1-FLAG_V-FLAG'},\n",
    "    {'id': 'skm20', 'Ab': 'RNAP', 'control': 'skm25', 'name': 'S1-FLAG_V-RNAP'},\n",
    "    {'id': 'skm21', 'Ab': 'S9.6', 'control': 'skm25', 'name': 'S1-FLAG_V-S9.6'},\n",
    "    \n",
    "    {'id': 'skm22', 'Ab': 'FLAG', 'control': 'skm26', 'name': 'S1-FLAG_rnh-FLAG'},\n",
    "    {'id': 'skm23', 'Ab': 'RNAP', 'control': 'skm26', 'name': 'S1-FLAG_rnh-RNAP'},\n",
    "    {'id': 'skm24', 'Ab': 'S9.6', 'control': 'skm26', 'name': 'S1-FLAG_rnh-S9.6'},\n",
    "]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HepFLAG-WTS1-FLAG\n",
      "HepFLAG-WTS1-RNAP\n",
      "HepFLAG-WTS1-S9.6\n",
      "HepFLAG-S1d56-FLAG\n",
      "HepFLAG-S1d56-RNAP\n",
      "HepFLAG-S1d56-S9.6\n",
      "S1d56-HepFLAG-FLAG\n",
      "S1d56-HepFLAG-RNAP\n",
      "S1d56-HepFLAG-S9.6\n",
      "S1FLAG-dHep-FLAG\n",
      "S1FLAG-dHep-RNAP\n",
      "S1FLAG-dHep-S9.6\n",
      "Hep-FLAG_V-FLAG\n",
      "Hep-FLAG_V-RNAP\n",
      "Hep-FLAG_V-S9.6\n",
      "Hep-FLAG_rnh-FLAG\n",
      "Hep-FLAG_rnh-RNAP\n",
      "Hep-FLAG_rnh-S9.6\n",
      "S1-FLAG_V-FLAG\n",
      "S1-FLAG_V-RNAP\n",
      "S1-FLAG_V-S9.6\n",
      "S1-FLAG_rnh-FLAG\n",
      "S1-FLAG_rnh-RNAP\n",
      "S1-FLAG_rnh-S9.6\n"
     ]
    }
   ],
   "source": [
    "for sample in samples:\n",
    "    name = sample['name']\n",
    "    !echo $name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 28 Oct 2016 10:38:59: \n",
      "# Command line: callpeak -t ../results/skm01/skm01_sorted.bam -c ../results/skm27/skm27_sorted.bam --name HepFLAG-WTS1-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm01\n",
      "# ARGUMENTS LIST:\n",
      "# name = HepFLAG-WTS1-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm01/skm01_sorted.bam']\n",
      "# control file = ['../results/skm27/skm27_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 10:38:59: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 10:38:59: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:39:10:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:39:22:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:39:33:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:39:44:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:39:55:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:40:06:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:40:17:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:40:28: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:40:39:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:40:50:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:41:01:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:41:13:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:41:24:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:41:35:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:41:46:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:41:58:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:09:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:22: #1 mean fragment size is determined as 236 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:22: #1 note: mean fragment size in control is 224 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:22: #1 fragment size = 236 \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:22: #1  total fragments in treatment: 7130191 \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:22: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:22: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:49: #1  fragments after filtering in treatment: 6962520 \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:49: #1  Redundant rate of treatment: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:49: #1  total fragments in control: 9179816 \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:49: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:42:49: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #1  fragments after filtering in control: 8987220 \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #2 Use 236 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 10:43:24: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:21: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:24: #4 Write output xls file... ../results/skm01/HepFLAG-WTS1-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:24: #4 Write peak in narrowPeak format file... ../results/skm01/HepFLAG-WTS1-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:24: #4 Write summits bed file... ../results/skm01/HepFLAG-WTS1-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:24: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:24: \n",
      "# Command line: callpeak -t ../results/skm02/skm02_sorted.bam -c ../results/skm27/skm27_sorted.bam --name HepFLAG-WTS1-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm02\n",
      "# ARGUMENTS LIST:\n",
      "# name = HepFLAG-WTS1-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm02/skm02_sorted.bam']\n",
      "# control file = ['../results/skm27/skm27_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:24: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:24: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:36:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:46:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:44:58:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:45:08:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:45:19:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:45:30:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:45:41:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:45:52:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:46:03:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:46:13:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:46:24:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:46:35:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:46:46:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:46:57:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:47:08:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:47:19:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:47:30:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:47:41:  18000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:48:05: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:48:16:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:48:27:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:48:39:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:48:50:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:12:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:23:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:34:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:46:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:59: #1 mean fragment size is determined as 211 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:59: #1 note: mean fragment size in control is 224 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:59: #1 fragment size = 211 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:59: #1  total fragments in treatment: 18176557 \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:59: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:49:59: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:04: #1  fragments after filtering in treatment: 16321756 \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:04: #1  Redundant rate of treatment: 0.10 \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:04: #1  total fragments in control: 9179816 \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:04: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:04: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #1  fragments after filtering in control: 8987220 \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #2 Use 211 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 10:51:37: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:38: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:41: #4 Write output xls file... ../results/skm02/HepFLAG-WTS1-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:41: #4 Write peak in narrowPeak format file... ../results/skm02/HepFLAG-WTS1-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:41: #4 Write summits bed file... ../results/skm02/HepFLAG-WTS1-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:41: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:41: \n",
      "# Command line: callpeak -t ../results/skm03/skm03_sorted.bam -c ../results/skm27/skm27_sorted.bam --name HepFLAG-WTS1-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm03\n",
      "# ARGUMENTS LIST:\n",
      "# name = HepFLAG-WTS1-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm03/skm03_sorted.bam']\n",
      "# control file = ['../results/skm27/skm27_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:41: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:41: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:54:52:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:55:04:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:55:15:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:55:26:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:55:38:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:55:49:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:56:01:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:56:13:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:56:30: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:56:41:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:56:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:57:04:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:57:15:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:57:26:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:57:38:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:57:49:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:00:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:12:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:25: #1 mean fragment size is determined as 195 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:25: #1 note: mean fragment size in control is 224 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:25: #1 fragment size = 195 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:25: #1  total fragments in treatment: 8554719 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:25: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:25: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:58: #1  fragments after filtering in treatment: 8044816 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:58: #1  Redundant rate of treatment: 0.06 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:58: #1  total fragments in control: 9179816 \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:58: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 10:58:58: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #1  fragments after filtering in control: 8987220 \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #2 Use 195 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 10:59:34: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:46: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:48: #4 Write output xls file... ../results/skm03/HepFLAG-WTS1-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:48: #4 Write peak in narrowPeak format file... ../results/skm03/HepFLAG-WTS1-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:48: #4 Write summits bed file... ../results/skm03/HepFLAG-WTS1-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:48: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:48: \n",
      "# Command line: callpeak -t ../results/skm04/skm04_sorted.bam -c ../results/skm28/skm28_sorted.bam --name HepFLAG-S1d56-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm04\n",
      "# ARGUMENTS LIST:\n",
      "# name = HepFLAG-S1d56-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm04/skm04_sorted.bam']\n",
      "# control file = ['../results/skm28/skm28_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:48: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 11:00:48: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:01:00:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:01:11:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:01:22:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:01:33:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:01:44:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:01:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:02:07:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:02:27: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:02:38:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:02:49:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:03:00:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:03:11:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:03:22:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:03:33:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:03:44:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:03:55:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:04:06:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:04:17:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:04:28:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:04:39:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:04:50:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:05:01:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:05:12:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:05:23:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:05:34:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:07:47: #1 mean fragment size is determined as 234 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 11:07:47: #1 note: mean fragment size in control is 193 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 11:07:47: #1 fragment size = 234 \n",
      "INFO  @ Fri, 28 Oct 2016 11:07:47: #1  total fragments in treatment: 7802276 \n",
      "INFO  @ Fri, 28 Oct 2016 11:07:47: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:07:47: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:08:14: #1  fragments after filtering in treatment: 7576873 \n",
      "INFO  @ Fri, 28 Oct 2016 11:08:14: #1  Redundant rate of treatment: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 11:08:14: #1  total fragments in control: 17426272 \n",
      "INFO  @ Fri, 28 Oct 2016 11:08:14: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:08:14: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #1  fragments after filtering in control: 16514262 \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #1  Redundant rate of control: 0.05 \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #2 Use 234 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 11:09:17: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:24: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:27: #4 Write output xls file... ../results/skm04/HepFLAG-S1d56-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:27: #4 Write peak in narrowPeak format file... ../results/skm04/HepFLAG-S1d56-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:27: #4 Write summits bed file... ../results/skm04/HepFLAG-S1d56-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:27: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:27: \n",
      "# Command line: callpeak -t ../results/skm05/skm05_sorted.bam -c ../results/skm28/skm28_sorted.bam --name HepFLAG-S1d56-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm05\n",
      "# ARGUMENTS LIST:\n",
      "# name = HepFLAG-S1d56-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm05/skm05_sorted.bam']\n",
      "# control file = ['../results/skm28/skm28_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:27: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:27: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:38:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:10:49:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:11:00:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:11:11:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:11:21:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:11:32:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:11:49: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:12:00:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:12:10:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:12:21:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:12:32:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:12:43:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:12:53:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:13:04:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:13:15:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:13:26:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:13:36:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:13:47:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:13:58:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:14:10:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:14:21:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:14:32:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:14:43:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:14:54:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:12: #1 mean fragment size is determined as 202 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:12: #1 note: mean fragment size in control is 193 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:12: #1 fragment size = 202 \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:12: #1  total fragments in treatment: 6845274 \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:12: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:12: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:38: #1  fragments after filtering in treatment: 6411198 \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:38: #1  Redundant rate of treatment: 0.06 \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:38: #1  total fragments in control: 17426272 \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:38: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:17:38: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #1  fragments after filtering in control: 16514262 \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #1  Redundant rate of control: 0.05 \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #2 Use 202 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 11:18:43: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:02: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:04: #4 Write output xls file... ../results/skm05/HepFLAG-S1d56-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:04: #4 Write peak in narrowPeak format file... ../results/skm05/HepFLAG-S1d56-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:04: #4 Write summits bed file... ../results/skm05/HepFLAG-S1d56-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:04: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:05: \n",
      "# Command line: callpeak -t ../results/skm06/skm06_sorted.bam -c ../results/skm28/skm28_sorted.bam --name HepFLAG-S1d56-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm06\n",
      "# ARGUMENTS LIST:\n",
      "# name = HepFLAG-S1d56-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm06/skm06_sorted.bam']\n",
      "# control file = ['../results/skm28/skm28_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:05: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:05: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:16:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:27:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:38:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:20:49:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:21:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:21:12:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:21:23:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:21:35:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:21:46:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:21:56:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:22:07:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:22:19:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:22:30:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:22:41:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:22:52:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:23:03:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:23:14:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:23:43: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:23:53:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:24:04:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:24:16:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:24:26:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:24:37:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:24:48:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:25:00:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:25:11:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:25:22:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:25:33:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:25:44:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:25:55:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:26:07:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:26:18:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:26:29:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:26:40:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:26:51:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:29:10: #1 mean fragment size is determined as 226 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 11:29:10: #1 note: mean fragment size in control is 193 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 11:29:10: #1 fragment size = 226 \n",
      "INFO  @ Fri, 28 Oct 2016 11:29:10: #1  total fragments in treatment: 17417517 \n",
      "INFO  @ Fri, 28 Oct 2016 11:29:10: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:29:10: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:30:14: #1  fragments after filtering in treatment: 15400654 \n",
      "INFO  @ Fri, 28 Oct 2016 11:30:14: #1  Redundant rate of treatment: 0.12 \n",
      "INFO  @ Fri, 28 Oct 2016 11:30:14: #1  total fragments in control: 17426272 \n",
      "INFO  @ Fri, 28 Oct 2016 11:30:14: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:30:14: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #1  fragments after filtering in control: 16514262 \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #1  Redundant rate of control: 0.05 \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #2 Use 226 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 11:31:18: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 11:33:58: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:01: #4 Write output xls file... ../results/skm06/HepFLAG-S1d56-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:01: #4 Write peak in narrowPeak format file... ../results/skm06/HepFLAG-S1d56-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:01: #4 Write summits bed file... ../results/skm06/HepFLAG-S1d56-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:01: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:01: \n",
      "# Command line: callpeak -t ../results/skm07/skm07_sorted.bam -c ../results/skm28/skm28_sorted.bam --name S1d56-HepFLAG-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm07\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1d56-HepFLAG-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm07/skm07_sorted.bam']\n",
      "# control file = ['../results/skm28/skm28_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:01: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:01: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:13:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:23:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:34:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:45:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:34:57:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:35:08:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:35:19:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:35:30:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:35:41:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:35:52:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:36:03:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:36:15:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:36:36: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:36:47:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:36:57:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:37:09:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:37:20:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:37:32:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:37:43:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:37:54:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:38:05:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:38:16:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:38:27:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:38:39:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:38:49:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:39:00:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:39:12:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:39:22:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:39:33:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:39:44:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:01: #1 mean fragment size is determined as 224 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:01: #1 note: mean fragment size in control is 193 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:01: #1 fragment size = 224 \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:01: #1  total fragments in treatment: 12547965 \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:01: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:01: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:46: #1  fragments after filtering in treatment: 12098514 \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:46: #1  Redundant rate of treatment: 0.04 \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:46: #1  total fragments in control: 17426272 \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:46: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:42:46: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #1  fragments after filtering in control: 16514262 \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #1  Redundant rate of control: 0.05 \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #2 Use 224 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 11:43:50: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:19: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:22: #4 Write output xls file... ../results/skm07/S1d56-HepFLAG-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:22: #4 Write peak in narrowPeak format file... ../results/skm07/S1d56-HepFLAG-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:22: #4 Write summits bed file... ../results/skm07/S1d56-HepFLAG-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:22: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:22: \n",
      "# Command line: callpeak -t ../results/skm08/skm08_sorted.bam -c ../results/skm28/skm28_sorted.bam --name S1d56-HepFLAG-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm08\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1d56-HepFLAG-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm08/skm08_sorted.bam']\n",
      "# control file = ['../results/skm28/skm28_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:22: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:22: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:34:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:45:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:45:56:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:46:08:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:46:19:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:46:30:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:46:42:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:46:53:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:47:13: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:47:24:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:47:35:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:47:47:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:47:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:48:09:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:48:20:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:48:31:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:48:42:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:48:53:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:49:04:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:49:15:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:49:26:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:49:36:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:49:47:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:49:58:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:50:10:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:50:21:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:52:32: #1 mean fragment size is determined as 227 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 11:52:32: #1 note: mean fragment size in control is 193 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 11:52:32: #1 fragment size = 227 \n",
      "INFO  @ Fri, 28 Oct 2016 11:52:32: #1  total fragments in treatment: 8866027 \n",
      "INFO  @ Fri, 28 Oct 2016 11:52:32: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:52:32: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:53:04: #1  fragments after filtering in treatment: 7909309 \n",
      "INFO  @ Fri, 28 Oct 2016 11:53:04: #1  Redundant rate of treatment: 0.11 \n",
      "INFO  @ Fri, 28 Oct 2016 11:53:04: #1  total fragments in control: 17426272 \n",
      "INFO  @ Fri, 28 Oct 2016 11:53:04: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:53:04: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #1  fragments after filtering in control: 16514262 \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #1  Redundant rate of control: 0.05 \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #2 Use 227 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 11:54:10: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:42: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:45: #4 Write output xls file... ../results/skm08/S1d56-HepFLAG-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:45: #4 Write peak in narrowPeak format file... ../results/skm08/S1d56-HepFLAG-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:45: #4 Write summits bed file... ../results/skm08/S1d56-HepFLAG-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:45: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:45: \n",
      "# Command line: callpeak -t ../results/skm09/skm09_sorted.bam -c ../results/skm28/skm28_sorted.bam --name S1d56-HepFLAG-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm09\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1d56-HepFLAG-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm09/skm09_sorted.bam']\n",
      "# control file = ['../results/skm28/skm28_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:45: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:45: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:55:56:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:56:07:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:56:17:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:56:28:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:56:39:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:56:50:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:57:01:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:57:12:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:57:23:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:57:35:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:57:51: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 11:58:01:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:58:12:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:58:24:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:58:35:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:58:46:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:58:56:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:59:08:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:59:19:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:59:31:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:59:41:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 11:59:52:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:00:03:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:00:14:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:00:25:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:00:36:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:00:47:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:00:58:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:13: #1 mean fragment size is determined as 205 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:13: #1 note: mean fragment size in control is 193 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:13: #1 fragment size = 205 \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:13: #1  total fragments in treatment: 10405619 \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:13: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:13: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:50: #1  fragments after filtering in treatment: 9818581 \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:50: #1  Redundant rate of treatment: 0.06 \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:50: #1  total fragments in control: 17426272 \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:50: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:03:50: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #1  fragments after filtering in control: 16514262 \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #1  Redundant rate of control: 0.05 \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #2 Use 205 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:04:52: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:33: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:36: #4 Write output xls file... ../results/skm09/S1d56-HepFLAG-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:36: #4 Write peak in narrowPeak format file... ../results/skm09/S1d56-HepFLAG-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:36: #4 Write summits bed file... ../results/skm09/S1d56-HepFLAG-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:36: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:36: \n",
      "# Command line: callpeak -t ../results/skm10/skm10_sorted.bam -c ../results/skm30/skm30_sorted.bam --name S1FLAG-dHep-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm10\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1FLAG-dHep-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm10/skm10_sorted.bam']\n",
      "# control file = ['../results/skm30/skm30_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:36: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:36: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:47:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:06:58:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:07:09:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:07:20: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:07:31:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:07:42:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:07:53:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:08:04:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:08:15:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:08:26:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:08:37:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:08:49:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:00:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:11:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:30: #1 mean fragment size is determined as 197 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:30: #1 note: mean fragment size in control is 179 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:30: #1 fragment size = 197 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:30: #1  total fragments in treatment: 3630129 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:30: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:30: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:43: #1  fragments after filtering in treatment: 3497365 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:43: #1  Redundant rate of treatment: 0.04 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:43: #1  total fragments in control: 10567605 \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:43: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:09:43: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #1  fragments after filtering in control: 10241642 \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #2 Use 197 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:10:21: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:06: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:08: #4 Write output xls file... ../results/skm10/S1FLAG-dHep-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:08: #4 Write peak in narrowPeak format file... ../results/skm10/S1FLAG-dHep-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:08: #4 Write summits bed file... ../results/skm10/S1FLAG-dHep-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:08: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:09: \n",
      "# Command line: callpeak -t ../results/skm11/skm11_sorted.bam -c ../results/skm30/skm30_sorted.bam --name S1FLAG-dHep-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm11\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1FLAG-dHep-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm11/skm11_sorted.bam']\n",
      "# control file = ['../results/skm30/skm30_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:09: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:09: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:20:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:30:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:41:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:11:51:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:12:02:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:12:13:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:12:23: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:12:34:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:12:45:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:12:56:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:13:07:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:13:18:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:13:29:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:13:40:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:13:50:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:01:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:12:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:31: #1 mean fragment size is determined as 218 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:31: #1 note: mean fragment size in control is 179 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:31: #1 fragment size = 218 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:31: #1  total fragments in treatment: 6242071 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:31: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:31: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:54: #1  fragments after filtering in treatment: 6110805 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:54: #1  Redundant rate of treatment: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:54: #1  total fragments in control: 10567605 \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:54: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:14:54: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #1  fragments after filtering in control: 10241642 \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #2 Use 218 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:15:33: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:18: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:20: #4 Write output xls file... ../results/skm11/S1FLAG-dHep-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:20: #4 Write peak in narrowPeak format file... ../results/skm11/S1FLAG-dHep-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:20: #4 Write summits bed file... ../results/skm11/S1FLAG-dHep-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:20: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:21: \n",
      "# Command line: callpeak -t ../results/skm12/skm12_sorted.bam -c ../results/skm30/skm30_sorted.bam --name S1FLAG-dHep-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm12\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1FLAG-dHep-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm12/skm12_sorted.bam']\n",
      "# control file = ['../results/skm30/skm30_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:21: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:21: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:32:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:43:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:16:54:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:17:06:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:17:17:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:17:28:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:17:39:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:17:50:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:18:01:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:18:13:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:18:23:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:18:35:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:18:46:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:18:57:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:19:08:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:19:19:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:19:30:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:19:40:  18000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:19:51:  19000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:20:03:  20000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:20:14:  21000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:20:25:  22000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:20:55: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:21:06:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:21:17:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:21:28:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:21:38:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:21:50:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:22:01:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:22:12:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:22:24:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:22:35:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:22:46:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:23:05: #1 mean fragment size is determined as 200 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:23:05: #1 note: mean fragment size in control is 179 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:23:05: #1 fragment size = 200 \n",
      "INFO  @ Fri, 28 Oct 2016 12:23:05: #1  total fragments in treatment: 22155163 \n",
      "INFO  @ Fri, 28 Oct 2016 12:23:05: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:23:05: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:24:25: #1  fragments after filtering in treatment: 18597519 \n",
      "INFO  @ Fri, 28 Oct 2016 12:24:25: #1  Redundant rate of treatment: 0.16 \n",
      "INFO  @ Fri, 28 Oct 2016 12:24:25: #1  total fragments in control: 10567605 \n",
      "INFO  @ Fri, 28 Oct 2016 12:24:25: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:24:25: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #1  fragments after filtering in control: 10241642 \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #2 Use 200 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:25:04: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:37: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:40: #4 Write output xls file... ../results/skm12/S1FLAG-dHep-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:40: #4 Write peak in narrowPeak format file... ../results/skm12/S1FLAG-dHep-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:40: #4 Write summits bed file... ../results/skm12/S1FLAG-dHep-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:40: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:40: \n",
      "# Command line: callpeak -t ../results/skm13/skm13_sorted.bam -c ../results/skm25/skm25_sorted.bam --name Hep-FLAG_V-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm13\n",
      "# ARGUMENTS LIST:\n",
      "# name = Hep-FLAG_V-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm13/skm13_sorted.bam']\n",
      "# control file = ['../results/skm25/skm25_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:40: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:40: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:27:51:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:28:02:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:28:13:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:28:24:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:28:36:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:28:47:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:28:56: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:09:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:20:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:32:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:44:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:57: #1 mean fragment size is determined as 248 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:57: #1 note: mean fragment size in control is 258 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:57: #1 fragment size = 248 \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:57: #1  total fragments in treatment: 6234906 \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:57: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:29:57: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:20: #1  fragments after filtering in treatment: 6098280 \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:20: #1  Redundant rate of treatment: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:20: #1  total fragments in control: 4705185 \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:20: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:20: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #1  fragments after filtering in control: 4611348 \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #2 Use 248 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:30:37: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:11: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:14: #4 Write output xls file... ../results/skm13/Hep-FLAG_V-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:14: #4 Write peak in narrowPeak format file... ../results/skm13/Hep-FLAG_V-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:14: #4 Write summits bed file... ../results/skm13/Hep-FLAG_V-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:14: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:14: \n",
      "# Command line: callpeak -t ../results/skm14/skm14_sorted.bam -c ../results/skm25/skm25_sorted.bam --name Hep-FLAG_V-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm14\n",
      "# ARGUMENTS LIST:\n",
      "# name = Hep-FLAG_V-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm14/skm14_sorted.bam']\n",
      "# control file = ['../results/skm25/skm25_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:14: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:14: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:25:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:36:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:47:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:31:57:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:32:08:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:32:19:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:32:29:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:32:40:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:32:50:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:33:01:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:33:11:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:33:22:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:33:32:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:33:43:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:33:53:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:34:15: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:34:26:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:34:37:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:34:48:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:34:59:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:35:12: #1 mean fragment size is determined as 214 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:35:12: #1 note: mean fragment size in control is 258 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:35:12: #1 fragment size = 214 \n",
      "INFO  @ Fri, 28 Oct 2016 12:35:12: #1  total fragments in treatment: 15481444 \n",
      "INFO  @ Fri, 28 Oct 2016 12:35:12: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:35:12: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:07: #1  fragments after filtering in treatment: 14121806 \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:07: #1  Redundant rate of treatment: 0.09 \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:07: #1  total fragments in control: 4705185 \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:07: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:07: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #1  fragments after filtering in control: 4611348 \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #2 Use 214 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:36:24: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:40: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:43: #4 Write output xls file... ../results/skm14/Hep-FLAG_V-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:43: #4 Write peak in narrowPeak format file... ../results/skm14/Hep-FLAG_V-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:43: #4 Write summits bed file... ../results/skm14/Hep-FLAG_V-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:43: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:43: \n",
      "# Command line: callpeak -t ../results/skm15/skm15_sorted.bam -c ../results/skm25/skm25_sorted.bam --name Hep-FLAG_V-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm15\n",
      "# ARGUMENTS LIST:\n",
      "# name = Hep-FLAG_V-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm15/skm15_sorted.bam']\n",
      "# control file = ['../results/skm25/skm25_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:43: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:43: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:37:54:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:38:04:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:38:15:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:38:25:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:38:36:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:38:47:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:38:58:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:39:08:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:39:19:  9000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:39:29:  10000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:39:40:  11000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:39:51:  12000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:40:02:  13000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:40:13:  14000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:40:24:  15000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:40:35:  16000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:40:45:  17000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:40:56:  18000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:41:07:  19000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:41:18:  20000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:41:29:  21000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:41:40:  22000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:41:51:  23000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:42:01:  24000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:42:12:  25000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:42:23:  26000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:42:34:  27000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:42:44:  28000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:42:55:  29000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:43:06:  30000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:43:17:  31000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:43:28:  32000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:43:39:  33000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:43:49:  34000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:44:00:  35000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:44:11:  36000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:44:22:  37000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:44:32:  38000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:44:43:  39000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:44:53:  40000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:45:04:  41000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:46:03: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:46:14:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:46:25:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:46:36:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:46:47:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:47:01: #1 mean fragment size is determined as 149 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:47:01: #1 note: mean fragment size in control is 258 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:47:01: #1 fragment size = 149 \n",
      "INFO  @ Fri, 28 Oct 2016 12:47:01: #1  total fragments in treatment: 41831192 \n",
      "INFO  @ Fri, 28 Oct 2016 12:47:01: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:47:01: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:36: #1  fragments after filtering in treatment: 36771239 \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:36: #1  Redundant rate of treatment: 0.12 \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:36: #1  total fragments in control: 4705185 \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:36: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:36: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #1  fragments after filtering in control: 4611348 \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #2 Use 149 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:49:53: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:10: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:13: #4 Write output xls file... ../results/skm15/Hep-FLAG_V-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:13: #4 Write peak in narrowPeak format file... ../results/skm15/Hep-FLAG_V-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:13: #4 Write summits bed file... ../results/skm15/Hep-FLAG_V-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:13: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:13: \n",
      "# Command line: callpeak -t ../results/skm16/skm16_sorted.bam -c ../results/skm26/skm26_sorted.bam --name Hep-FLAG_rnh-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm16\n",
      "# ARGUMENTS LIST:\n",
      "# name = Hep-FLAG_rnh-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm16/skm16_sorted.bam']\n",
      "# control file = ['../results/skm26/skm26_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:13: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:13: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:24:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:35:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:46:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:50:57:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:51:08:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:51:19:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:51:30:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:51:46: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:51:57:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:52:09:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:52:20:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:52:30:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:52:42:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:52:53:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:04:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:15:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:30: #1 mean fragment size is determined as 249 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:30: #1 note: mean fragment size in control is 261 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:30: #1 fragment size = 249 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:30: #1  total fragments in treatment: 7521252 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:30: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:30: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:58: #1  fragments after filtering in treatment: 7338696 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:58: #1  Redundant rate of treatment: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:58: #1  total fragments in control: 8384087 \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:58: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:53:58: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #1  fragments after filtering in control: 8168720 \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #2 Use 249 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:54:30: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:18: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:20: #4 Write output xls file... ../results/skm16/Hep-FLAG_rnh-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:20: #4 Write peak in narrowPeak format file... ../results/skm16/Hep-FLAG_rnh-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:20: #4 Write summits bed file... ../results/skm16/Hep-FLAG_rnh-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:20: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:21: \n",
      "# Command line: callpeak -t ../results/skm17/skm17_sorted.bam -c ../results/skm26/skm26_sorted.bam --name Hep-FLAG_rnh-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm17\n",
      "# ARGUMENTS LIST:\n",
      "# name = Hep-FLAG_rnh-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm17/skm17_sorted.bam']\n",
      "# control file = ['../results/skm26/skm26_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:21: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:21: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:32:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:42:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:55:54:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:56:05:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:56:16:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:56:24: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:56:35:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:56:46:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:56:58:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:57:08:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:57:20:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:57:31:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:57:42:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:57:53:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:09: #1 mean fragment size is determined as 207 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:09: #1 note: mean fragment size in control is 261 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:09: #1 fragment size = 207 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:09: #1  total fragments in treatment: 5018292 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:09: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:09: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:27: #1  fragments after filtering in treatment: 4829526 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:27: #1  Redundant rate of treatment: 0.04 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:27: #1  total fragments in control: 8384087 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:27: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:27: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #1  fragments after filtering in control: 8168720 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #2 Use 207 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 12:58:58: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:47: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:50: #4 Write output xls file... ../results/skm17/Hep-FLAG_rnh-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:50: #4 Write peak in narrowPeak format file... ../results/skm17/Hep-FLAG_rnh-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:50: #4 Write summits bed file... ../results/skm17/Hep-FLAG_rnh-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:50: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:50: \n",
      "# Command line: callpeak -t ../results/skm18/skm18_sorted.bam -c ../results/skm26/skm26_sorted.bam --name Hep-FLAG_rnh-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm18\n",
      "# ARGUMENTS LIST:\n",
      "# name = Hep-FLAG_rnh-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm18/skm18_sorted.bam']\n",
      "# control file = ['../results/skm26/skm26_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:50: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 12:59:50: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:00:01:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:00:12:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:00:24:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:00:35:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:00:46:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:00:57:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:01:17: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:01:28:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:01:39:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:01:51:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:02:02:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:02:13:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:02:24:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:02:35:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:02:46:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:01: #1 mean fragment size is determined as 220 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:01: #1 note: mean fragment size in control is 261 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:01: #1 fragment size = 220 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:01: #1  total fragments in treatment: 6940997 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:01: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:01: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:27: #1  fragments after filtering in treatment: 6693574 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:27: #1  Redundant rate of treatment: 0.04 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:27: #1  total fragments in control: 8384087 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:27: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:27: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #1  fragments after filtering in control: 8168720 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #2 Use 220 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 13:03:59: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:50: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:53: #4 Write output xls file... ../results/skm18/Hep-FLAG_rnh-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:53: #4 Write peak in narrowPeak format file... ../results/skm18/Hep-FLAG_rnh-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:53: #4 Write summits bed file... ../results/skm18/Hep-FLAG_rnh-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:53: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:53: \n",
      "# Command line: callpeak -t ../results/skm19/skm19_sorted.bam -c ../results/skm25/skm25_sorted.bam --name S1-FLAG_V-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm19\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1-FLAG_V-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm19/skm19_sorted.bam']\n",
      "# control file = ['../results/skm25/skm25_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:53: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 13:04:53: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:05:04:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:05:15:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:05:27:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:05:38:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:05:49:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:05:56: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:08:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:19:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:30:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:41:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:55: #1 mean fragment size is determined as 228 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:55: #1 note: mean fragment size in control is 258 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:55: #1 fragment size = 228 \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:55: #1  total fragments in treatment: 5067545 \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:55: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:06:55: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:13: #1  fragments after filtering in treatment: 4939020 \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:13: #1  Redundant rate of treatment: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:13: #1  total fragments in control: 4705185 \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:13: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:13: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #1  fragments after filtering in control: 4611348 \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #2 Use 228 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:31: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:55: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:58: #4 Write output xls file... ../results/skm19/S1-FLAG_V-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:58: #4 Write peak in narrowPeak format file... ../results/skm19/S1-FLAG_V-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:58: #4 Write summits bed file... ../results/skm19/S1-FLAG_V-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:58: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:58: \n",
      "# Command line: callpeak -t ../results/skm20/skm20_sorted.bam -c ../results/skm25/skm25_sorted.bam --name S1-FLAG_V-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm20\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1-FLAG_V-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm20/skm20_sorted.bam']\n",
      "# control file = ['../results/skm25/skm25_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:58: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 13:07:58: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:08:09:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:08:20:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:08:31:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:08:42:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:08:55: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:06:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:18:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:29:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:40:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:54: #1 mean fragment size is determined as 219 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:54: #1 note: mean fragment size in control is 258 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:54: #1 fragment size = 219 \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:54: #1  total fragments in treatment: 4776200 \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:54: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:09:54: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:11: #1  fragments after filtering in treatment: 4607520 \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:11: #1  Redundant rate of treatment: 0.04 \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:11: #1  total fragments in control: 4705185 \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:11: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:11: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #1  fragments after filtering in control: 4611348 \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #2 Use 219 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 13:10:28: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:06: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:08: #4 Write output xls file... ../results/skm20/S1-FLAG_V-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:08: #4 Write peak in narrowPeak format file... ../results/skm20/S1-FLAG_V-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:08: #4 Write summits bed file... ../results/skm20/S1-FLAG_V-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:08: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:09: \n",
      "# Command line: callpeak -t ../results/skm21/skm21_sorted.bam -c ../results/skm25/skm25_sorted.bam --name S1-FLAG_V-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm21\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1-FLAG_V-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm21/skm21_sorted.bam']\n",
      "# control file = ['../results/skm25/skm25_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:09: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:09: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:20:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:30:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:42:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:11:52:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:12:03:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:12:13:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:12:22: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:12:33:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:12:44:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:12:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:05:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:19: #1 mean fragment size is determined as 233 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:19: #1 note: mean fragment size in control is 258 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:19: #1 fragment size = 233 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:19: #1  total fragments in treatment: 6101669 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:19: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:19: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:41: #1  fragments after filtering in treatment: 5810114 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:41: #1  Redundant rate of treatment: 0.05 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:41: #1  total fragments in control: 4705185 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:41: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:41: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #1  fragments after filtering in control: 4611348 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #1  Redundant rate of control: 0.02 \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #2 Use 233 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 13:13:58: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:45: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:48: #4 Write output xls file... ../results/skm21/S1-FLAG_V-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:48: #4 Write peak in narrowPeak format file... ../results/skm21/S1-FLAG_V-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:48: #4 Write summits bed file... ../results/skm21/S1-FLAG_V-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:48: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:49: \n",
      "# Command line: callpeak -t ../results/skm22/skm22_sorted.bam -c ../results/skm26/skm26_sorted.bam --name S1-FLAG_rnh-FLAG --format BAMPE --gsize 4.6e6 --outdir ../results/skm22\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1-FLAG_rnh-FLAG\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm22/skm22_sorted.bam']\n",
      "# control file = ['../results/skm26/skm26_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:49: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:49: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:14:59:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:15:10:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:15:21:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:15:32:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:15:42:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:15:55: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:16:06:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:16:16:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:16:27:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:16:38:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:16:49:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:16:59:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:10:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:21:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:36: #1 mean fragment size is determined as 238 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:36: #1 note: mean fragment size in control is 261 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:36: #1 fragment size = 238 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:36: #1  total fragments in treatment: 5478019 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:36: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:36: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:56: #1  fragments after filtering in treatment: 5334379 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:56: #1  Redundant rate of treatment: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:56: #1  total fragments in control: 8384087 \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:56: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:17:56: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #1  fragments after filtering in control: 8168720 \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #2 Use 238 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 13:18:26: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:02: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: #4 Write output xls file... ../results/skm22/S1-FLAG_rnh-FLAG_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: #4 Write peak in narrowPeak format file... ../results/skm22/S1-FLAG_rnh-FLAG_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: #4 Write summits bed file... ../results/skm22/S1-FLAG_rnh-FLAG_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: \n",
      "# Command line: callpeak -t ../results/skm23/skm23_sorted.bam -c ../results/skm26/skm26_sorted.bam --name S1-FLAG_rnh-RNAP --format BAMPE --gsize 4.6e6 --outdir ../results/skm23\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1-FLAG_rnh-RNAP\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm23/skm23_sorted.bam']\n",
      "# control file = ['../results/skm26/skm26_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:05: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:16:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:27:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:38:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:19:50:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:12:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:22:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:33:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1 mean fragment size is determined as 210 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1 note: mean fragment size in control is 261 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1 fragment size = 210 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1  total fragments in treatment: 52 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1  fragments after filtering in treatment: 52 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1  Redundant rate of treatment: 0.00 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1  total fragments in control: 8384087 \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:20:48: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #1  fragments after filtering in control: 8168720 \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #2 Use 210 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:17: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:32: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:33: #4 Write output xls file... ../results/skm23/S1-FLAG_rnh-RNAP_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:33: #4 Write peak in narrowPeak format file... ../results/skm23/S1-FLAG_rnh-RNAP_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:33: #4 Write summits bed file... ../results/skm23/S1-FLAG_rnh-RNAP_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:33: Done! \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:34: \n",
      "# Command line: callpeak -t ../results/skm24/skm24_sorted.bam -c ../results/skm26/skm26_sorted.bam --name S1-FLAG_rnh-S9.6 --format BAMPE --gsize 4.6e6 --outdir ../results/skm24\n",
      "# ARGUMENTS LIST:\n",
      "# name = S1-FLAG_rnh-S9.6\n",
      "# format = BAMPE\n",
      "# ChIP-seq file = ['../results/skm24/skm24_sorted.bam']\n",
      "# control file = ['../results/skm26/skm26_sorted.bam']\n",
      "# effective genome size = 4.60e+06\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 1000 bps and 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:34: #1 read fragment files... \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:34: #1 read treatment fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:45:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:21:56:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:22:07:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:22:18:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:22:30:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:22:50: #1.2 read input fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:23:00:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:23:11:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:23:22:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:23:32:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:23:44:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:23:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:06:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:17:  8000000 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:32: #1 mean fragment size is determined as 239 bp from treatment \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:32: #1 note: mean fragment size in control is 261 bp -- value ignored \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:32: #1 fragment size = 239 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:32: #1  total fragments in treatment: 5927291 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:32: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:32: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:53: #1  fragments after filtering in treatment: 5505006 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:53: #1  Redundant rate of treatment: 0.07 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:53: #1  total fragments in control: 8384087 \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:53: #1 user defined the maximum fragments... \n",
      "INFO  @ Fri, 28 Oct 2016 13:24:53: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #1  fragments after filtering in control: 8168720 \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #1  Redundant rate of control: 0.03 \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #2 Use 239 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2016 13:25:23: #3 Pre-compute pvalue-qvalue table... \n",
      "INFO  @ Fri, 28 Oct 2016 13:26:22: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2016 13:26:25: #4 Write output xls file... ../results/skm24/S1-FLAG_rnh-S9.6_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2016 13:26:25: #4 Write peak in narrowPeak format file... ../results/skm24/S1-FLAG_rnh-S9.6_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2016 13:26:25: #4 Write summits bed file... ../results/skm24/S1-FLAG_rnh-S9.6_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2016 13:26:25: Done! \n"
     ]
    }
   ],
   "source": [
    "dir_tpl = '../results/{id}/{id}_sorted.bam'\n",
    "\n",
    "for sample in samples:\n",
    "    treatment = dir_tpl.format(id=sample['id'])\n",
    "    control = dir_tpl.format(id=sample['control'])\n",
    "    res_dir = '../results/{}'.format(sample['id'])\n",
    "    name = sample['name']\n",
    "    !macs2 callpeak \\\n",
    "        -t $treatment \\\n",
    "        -c $control \\\n",
    "        --name \"$name\" \\\n",
    "        --format BAMPE \\\n",
    "        --gsize 4.6e6 \\\n",
    "        --outdir $res_dir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
